inc/3rd-party/nextgen-gallery/inc/classes/class-imagify-ngg-attachment.php

```diff
@@ -56,23 +65,34 @@ class Imagify_NGG_Attachment extends Imagify_Attachment {
 	 */
 	public function __construct( $id ) {
 		if ( is_object( $id ) ) {
-			$this->image = $id;
-			$this->id    = (int) $id->pid;
+			if ( is_a( $id, 'nggImage' ) ) {
```

Comment:
You could use `instanceOf` insteand of `is_a()` which is somewhat more performant

`$id instanceOf nggImage`

Reply:
OK I didn't know about that, I'll do it.